cpp/include/cudf/detail/aggregation/aggregation.cuh

```diff
@@ -69,6 +69,18 @@ struct corresponding_operator<aggregation::SUM_OF_SQUARES> {
   using type = DeviceSum;
 };
 template <>
+struct corresponding_operator<aggregation::MEAN> {
+  using type = DeviceSum;
+};
+template <>
+struct corresponding_operator<aggregation::STD> {
+  using type = DeviceSum;
+};
+template <>
+struct corresponding_operator<aggregation::VARIANCE> {
+  using type = DeviceSum;
+};
+template <>
```

Comment:
This looks fishy to me. `MEAN` never needed a `corresponding_operator` before. Why is it necessary now?

Reply:
For initializing with identity. all hash aggregations sparse_results columns are initialized with identity.

Reply:
At the point of initializing with the identity, compound aggs should be preprocessed into their simple aggs, yes?

Reply:
removed